pelican/readers.py

```diff
@@ -233,7 +233,13 @@ def read(self, source_path):
         """Parse content and metadata of markdown files"""
 
         self._source_path = source_path
-        self._md = Markdown(extensions=self.extensions)
+        extension_configs = {}
+        if 'markdown.extensions.codehilite' in self.extensions:
+            extension_configs['markdown.extensions.codehilite'] = {
+                'css_class':'highlight'
+            }
+        self._md = Markdown(extensions=self.extensions,
+                            extension_configs=extension_configs)
```

Comment:
I don't think this will surprise too many unweary users. If they want some other `css_class`, they will have had something else in `MD_EXTENSIONS` anyway.